mv-openapi.yaml

```diff
@@ -0,0 +1,476 @@
+        required: true
+        schema:
+          type: string
+      - in: path
+        name: ballot-id
+        required: true
+        schema:
+          type: string
+    get:
+      operationId: get-ballot
+      summary: Gets a ballot by ID
+      responses:
+        '200':
+          description: A ballot object
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ballot'
+    delete:
+      operationId: delete-ballot
+      summary: Deletes a ballot
+      responses:
+        '200':
+          description: OK
+        '404':
+          $ref: '#/components/responses/ballot-not-found'
+      security:
+        - participantAuth: []
+  '/polls/{poll-id}/participants':
+    parameters:
+      - in: path
+        name: poll-id
+        required: true
+        schema:
+          type: string
+    post:
+      operationId: add-participant-to-poll
+      description: Add a participant to a poll
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/participant'
+      responses:
+        '200':
+          description: Invitation was sent
+          content:
+            text/plain:
+              schema:
+                type: string
+        '404':
+          $ref: '#/components/responses/poll-not-found'
+  '/polls/{poll-id}/participants/{participant-id}/':
+    parameters:
+      - in: path
+        name: participant-id
+        required: true
+        schema:
+          type: string
+    get:
+      operationId: get-poll-participant
+      summary: Get a participant from a poll
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/participant'
+        '404':
+          $ref: '#/components/responses/participant-not-found'
+      security:
+        - {}
+        - pollAuth: []
+    delete:
+      operationId: delete-poll-participant
+      summary: Delete a participant from a poll
+      responses:
+        '200':
+          description: OK
+        '404':
+          $ref: '#/components/responses/participant-not-found'
+      security:
+        - pollAuth: []
+  '/polls/{poll-id}/results':
+    parameters:
+      - in: path
+        name: poll-id
+        required: true
+        schema:
+          type: string
+    get:
+      operationId: get-poll-results
+      description: Get results of a poll
+      responses:
+        '200':
+          description: OK
+          content:
+            text/plain:
+              schema:
+                $ref: '#/components/schemas/result'
+        '404':
+          $ref: '#/components/responses/poll-not-found'
+components:
+  securitySchemes:
+    participantAuth:
+      type: http
+      scheme: bearer
+    pollAuth:
+      type: http
+      scheme: bearer
+    resultAuth:
+      type: http
+      scheme: bearer
+  schemas:
+    ballot:
```

Comment:
I read somewhere that schemas should start with an uppercase in openapi

Reply:
Ok I will investigate.

Reply:
Hum weird I can't find it. Then it is certainly not a super important convention!